### PR TITLE
Pop/up

### PR DESCRIPTION
**Pull request by Akshay-Jak**
### Functionality:
Improved Pop-Up alerts

### Solution:
Improved pop-up alerts for log-work, edit-task and edit functions in the profile component

### Risk level:
- [ ] high 
- [x] medium
- [ ] low

### How to test:
ng serve
firebase emulators:start

Screenshot (26)

### CODE REVIEW

File: src/app/body/profile/edit-profile/edit-profile.component.html [hunk 1]
```diff
@@ -42,14 +42,27 @@ <h3 class="modal-title" id="exampleModalLabel">Edit Profile</h3>
...
-                    <div class="row">
-                        <div class="col">Successfully Updated!</div>
+                    <div class="row mx-5">
+                        <div class="col-md-4"></div>
+                        <div class="col-md-4">
+                            <mat-icon aria-hidden="false" id="SuccessIcon">check_circle_outline</mat-icon>
+                        </div>
+                        <div class="col-md-4"></div>
                     </div>
+                    <div class="row"></div>
```
Comment (viv9k) [inline src/app/body/profile/edit-profile/edit-profile.component.html:52]: what is this row?

File: src/app/body/profile/edit-projects/edit-projects.component.html [hunk 2]
```diff
@@ -40,14 +40,27 @@ <h2 class="modal-title" id="exampleModalLabel">Edit Project</h2>
...
-                    <div class="row">
-                        <div class="col">Successfully Updated!</div>
+                    <div class="row mx-5">
+                        <div class="col-md-4"></div>
+                        <div class="col-md-4">
+                            <mat-icon aria-hidden="false" id="SuccessIcon">check_circle_outline</mat-icon>
+                        </div>
+                        <div class="col-md-4"></div>
                     </div>
+                    <div class="row"></div>
```
Comment (viv9k) [inline src/app/body/profile/edit-projects/edit-projects.component.html:50]: Please remove this extra row

**Review by viv9k — CHANGES_REQUESTED** [review]
Please do the required changes and update

File: src/app/body/profile/edit-projects/edit-projects.component.html (hunk 2 above)
Comment (Akshay-Jak) [reply src/app/body/profile/edit-projects/edit-projects.component.html]: removed extra rows

File: src/app/body/profile/edit-profile/edit-profile.component.html [hunk 3]
```diff
@@ -42,14 +42,27 @@ <h3 class="modal-title" id="exampleModalLabel">Edit Profile</h3>
...
-                    <div class="row">
-                        <div class="col">Successfully Updated!</div>
+                    <div class="row mx-5">
+                        <div class="col-md-4"></div>
+                        <div class="col-md-4">
+                            <mat-icon aria-hidden="false" id="SuccessIcon">check_circle_outline</mat-icon>
+                        </div>
+                        <div class="col-md-4"></div>
                     </div>
+                    <!-- <div class="row"></div> -->
```
Comment (k0malSharma) [inline src/app/body/profile/edit-profile/edit-profile.component.html:52]: Remove Comment

**Comment by viv9k** [comment]
This branch can we merged after my footer changes

**Review by viv9k — APPROVED** [review]
Approved

**Pull request merged**